src/github.com/getlantern/flashlight/flashlight.go

```diff
@@ -381,12 +394,11 @@ func applyClientConfig(client *client.Client, cfg *config.Config) {
 
 	// We offload this onto a go routine because creating the http clients
 	// blocks on waiting for the local server, and the local server starts
-	// on the thread, otherwise creating a deadlock.
+	// later on this same thread, so it would otherwise creating a deadlock.
 	go func() {
-		withHttpClient(cfg.Addr, config.Configure)
-		withHttpClient(cfg.Addr, geolookup.Configure)
 		withHttpClient(cfg.Addr, statserver.Configure)
 	}()
+
 }
 
 func withHttpClient(addr string, withClient func(client *http.Client)) {
```

Comment:
`statserver` uses the http client to lookup the IP address of other clients, though, not us. So `pubsub` won't work I don't think.